<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-MD70B7S" failures="1" tests="7" name="Test" time="0.454" errors="0" timestamp="2023-08-15T14:54:11 IST">
  <testcase classname="TestNG.Hard_assert" name="Hard" time="0.06"/>
  <testcase classname="TestNG.Hard_assert" name="Hard_assertt" time="0.046">
    <skipped/>
  </testcase> <!-- Hard_assertt -->
  <testcase classname="TestNG.Hard_assert" name="Hard_assertt" time="0.02">
    <skipped/>
  </testcase> <!-- Hard_assertt -->
  <testcase classname="TestNG.Hard_assert" name="Hard_assertt" time="0.01">
    <skipped/>
  </testcase> <!-- Hard_assertt -->
  <testcase classname="TestNG.Hard_assert" name="Hard_assertt" time="0.01">
    <skipped/>
  </testcase> <!-- Hard_assertt -->
  <testcase classname="TestNG.Hard_assert" name="Hard_assertt" time="0.009">
    <skipped/>
  </testcase> <!-- Hard_assertt -->
  <testcase classname="TestNG.Hard_assert" name="Hard_assertt" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [verifi] but found [verification]">
      <![CDATA[java.lang.AssertionError: expected [verifi] but found [verification]
at TestNG.Hard_assert.Hard_assertt(Hard_assert.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- Hard_assertt -->
</testsuite> <!-- Test -->
